### ISSUE TITLE
Feauture requst: Possible to add proxy-support?

### ISSUE THREAD

**Issue by qnorsten** [spaam/svtplay-dl#428]
Hi!
Is it possible to add some sort of -proxy command to bypass geoblocks. 
Was trying to set up some cron jobs on a server I got, but unfortunatly it is located in France, so most of the content is geoblocked. 

If it is not possible to add above, does anyone else know of any way to run just svtplay-dl through a proxy using some other tools? I have access to swedish proxy-servers and servers in Sweden to pass an ssh-tunnel through, but have not found a way to only pass svtplay-dl traffic through.

**Comment by spaam** [comment]
http://stackoverflow.com/questions/11726881/how-to-set-an-http-proxy-in-python-2-7 have you tried that?

**Comment by qnorsten** [comment]
Thanks spaam, will try that later. 
I also found out that --http-header "X-FORWARDED-FOR=" seems to work fine on SVTplay atleast (like you had mentioned in an older issue)

**Comment by spaam** [comment]
yeah. i guess i should document those "hacks" on the webpage or something.

**Comment by qnorsten** [comment]
Yea that might be a good idea. 
Is there anymore interesting "hacks"?

**Comment by robertfoss** [comment]
Actually SOCKS support would be rather helpful for those of us living abroad.

**Comment by forthrin** [comment]
Can anyone provide a command line that will run the current version of `svtplay-dl` through a localhost proxy (127.0.0.1:XXXX)?

**Comment by forthrin** [comment]
Does anyone know how to run a proxy only on a select few domains? All browsers (and macOS system settings) are based the opposite way: "Run proxy on all domains *except* a select few domains".

I tried using an Automatic Proxy Configuration (proxy.pac file), but it does not work! Can anyone help?

    function FindProxyForURL(url, host) {

      if(dnsDomainIs(host, "www.svtplay.se"))
        return "SOCKS 127.0.0.1:9050";
    
      return "DIRECT";
    
    }

Some pitfalls I ran into:

- PAC files must lie on remote host, not file:// or http://localhost/
- PAC files must be renamed to make client notice changes
- PAC files can't be debugged since alert() is deprecated
